mover-rclone/active.sh

```diff
@@ -22,11 +19,11 @@ RCLONE_FLAGS=(--checksum --one-file-system --create-empty-src-dirs --progress --
 START_TIME=$SECONDS
 case "${DIRECTION}" in
 source)
-    rclone --config "${RCLONE_CONFIG}" sync "${RCLONE_FLAGS[@]}" /data "${RCLONE_CONFIG_SECTION}:${RCLONE_DEST_PATH}"
+    rclone copy "${RCLONE_FLAGS[@]}" "${MOUNT_PATH}" "${RCLONE_CONFIG_SECTION}:${RCLONE_DEST_PATH}"
```

Comment:
Are you sure you want `copy` and not `sync`? It won't remove deleted files, so the destination will eventually end up with all files that have ever existed.